File: Exercício Repetição 1/Exercício Repetição 1.py
opcao=0

while opcao!=3:

    print("MENU INICIAL")
    print("[1]IMCF")
    print("[2]PGCF")
    print("[3]SAIR DO PROGRAMA")
    opcao=int(input("INSIRA A OPÇÃO DESEJADA:"))
    
    
    if opcao == 1:
        print("IMCF")

        massa = float(input("Digite a massa do felino: "))
        comprimento_coluna =float(input("Digite o comprimento da coluna:"))

        imcf=massa/(comprimento_coluna * comprimento_coluna)
        print(f"O IMCF do gato é{imcf:.2f}.")

        if(imcf<8):
            print("O felino esta abaixo do peso.")
        elif(imcf>=8 and imcf<12):
            print("O felino esta no peso ideal.")
        else:
            print("O felino esta acima do peso.")

    elif opcao == 2:
        print("PGCF")

        ct=float(input("Digite a circunferência tóracica em cm:"))
        dcp=float(input("Digite a distância entre o calcâneo e a patela em cm:"))

        pgcf=(((ct/0.7067)-dcp)/0.9156)-dcp
        print(f"O percentual de gordura corporal felina é {pgcf:.2f}%")

        if(pgcf<20):
            print("O felino esta abaixo do peso.")
        elif(pgcf>=20 and pgcf<27):
            print("O felino esta no peso ideal.")
        else:
            print("O felino esta acima do peso.")

    elif opcao == 3:
        print("Programa encerrado.")
        
    else:
        print("ERRO! Digite um valor valido!")

File: Exercício Repetição 1/Exercício Repetição 1.c
#include <stdio.h>

int main()
{
    int opcao = 0;

    while (opcao != 3)
    {
        printf("MENU INICIAL\n");
        printf("[1] IMCF\n");
        printf("[2] PGCF\n");
        printf("[3] SAIR DO PROGRAMA\n");
        printf("INSIRA A OPCAO DESEJADA:\n");
        scanf("%i", &opcao);

        switch (opcao)
        {
        case 1:
            printf("IMCF\n");
            float massa, comprimento_coluna, imcf;

            printf("Digite a massa do felino: \n");
            scanf("%f", &massa);
            printf("Digite o comprimento da coluna:\n");
            scanf("%f", &comprimento_coluna);

            imcf = massa / (comprimento_coluna * comprimento_coluna);
            printf("O IMCF do gato e %.2f.\n", imcf);

            if (imcf < 8)
                printf("O felino esta abaixo do peso.\n");
            else if (imcf >= 8 && imcf < 12)
                printf("O felino esta no peso ideal.\n");

            else
                printf("O felino esta acima do peso.\n");
            break;

        case 2:
            float ct, dcp, pgcf;
            printf("PGCF\n");
            printf("Digite a circunferencia toracica em cm:\n");
            scanf("%f", &ct);
            printf("Digite a distância entre o calcaneo e a patela em cm:\n");
            scanf("%f", &dcp);

            pgcf = (((ct / 0.7067) - dcp) / 0.9156) - dcp;
            printf("O percentual de gordura corporal felina e %.2f%%\n", pgcf);

            if (pgcf < 20)
                printf("O felino esta abaixo do peso.\n");

            else if (pgcf >= 20 && pgcf < 27)
                printf("O felino esta no peso ideal.\n");

            else
                printf("O felino esta acima do peso.\n");
            break;

        case 3:
            opcao = 3;
            printf("Programa encerrado.\n");
            break;

        default:
            printf("ERRO! Digite um valor valido!\n");
            break;
        }
    }
}
